hack/update-codegen.sh

```diff
@@ -41,7 +41,7 @@ EXTERNAL_INFORMER_PKG="k8s.io/client-go/informers" \
   ${REPO_ROOT_DIR}/hack/generate-knative.sh "injection" \
     k8s.io/client-go \
     k8s.io/api \
-    "admissionregistration:v1beta1 apps:v1 autoscaling:v1,v2beta1 batch:v1,v1beta1 core:v1 rbac:v1" \
+    "admissionregistration:v1beta1,v1 apps:v1 autoscaling:v1,v2beta1 batch:v1,v1beta1 core:v1 rbac:v1" \
```

Comment:
do we still need the beta?

Reply:
No - but I don't want to break anyone consuming it downstream so I left it

Reply:
We can drop with when the API version is dropped by K8s or it becomes a pain to maintain

Reply:
it's probably file. It's just more files :) If they're unused might not even be linked, so should not matter